requirements/_requirements.txt

```diff
@@ -10,7 +10,7 @@ prometheus-fastapi-instrumentator<=6.0.0
 redis<6.0.0
 requests>=2.26.0
 rich<=13.5.2
-supervision>=0.20.0,<1.0.0
+supervision>=0.20.0,<=0.21.0
```

Comment:
this change seems to be not related with the florence2 aliases, do we need this change here?

Reply:
As I said in the description of this pr:

> Looks like supervision removed from_roboflow so I had to pin supervision low for these tests to pass

I have a separate pr fixing the issues in the tests so we can remove the pin

Reply:
I understand but if we merge this change the #526 is not changing pin in requirements

Reply:
Ah yep it's merged, I'll merge master in here and remove the requirements change